Shaders/Technicolor.fx

```diff
@@ -21,12 +21,12 @@ uniform float Strength <
 
 float3 TechnicolorPass(float4 vpos : SV_Position, float2 texcoord : TexCoord) : SV_Target
 {
-	const float3 cyanfilter = float3(0.0, 1.30, 1.0);
-	const float3 magentafilter = float3(1.0, 0.0, 1.05);
-	const float3 yellowfilter = float3(1.6, 1.6, 0.05);
-	const float2 redorangefilter = float2(1.05, 0.620); // RG_
-	const float2 greenfilter = float2(0.30, 1.0);       // RG_
-	const float2 magentafilter2 = magentafilter.rb;     // R_B
+	static const float3 cyanfilter = float3(0.0, 1.30, 1.0);
+	static const float3 magentafilter = float3(1.0, 0.0, 1.05);
+	static const float3 yellowfilter = float3(1.6, 1.6, 0.05);
+	static const float2 redorangefilter = float2(1.05, 0.620); // RG_
+	static const float2 greenfilter = float2(0.30, 1.0);       // RG_
+	static const float2 magentafilter2 = magentafilter.rb;     // R_B
```

Comment:
This code is equivalent. The `static` keyword has no effect in function scopes on the generated byte code. Proof:

------------
```hlsl
float4 main() : COLOR
{
	const float3 test = float3(4.0, 3.0, 2.0);

	return float4(test, 1.0);
}
```
==>
```
//
// Generated by Microsoft (R) HLSL Shader Compiler 10.1
    ps_3_0
    def c0, 4, 3, 2, 1
    mov oC0, c0

// approximately 1 instruction slot used
```
------------
```hlsl
float4 main() : COLOR
{
	static const float3 test = float3(4.0, 3.0, 2.0);

	return float4(test, 1.0);
}
```
==>
```
//
// Generated by Microsoft (R) HLSL Shader Compiler 10.1
    ps_3_0
    def c0, 4, 3, 2, 1
    mov oC0, c0

// approximately 1 instruction slot used
```

Reply:
Ok, were can i see the compiled shader code? To avoid such things in future.
I used to GPU ShaderAnalyzer 1.59, but maybe you know better way?